modules/Microsoft.ContainerService/managedClusters/agentPools/deploy.bicep

```diff
@@ -187,11 +187,11 @@ resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (ena
   }
 }
 
-resource managedCluster 'Microsoft.ContainerService/managedClusters@2021-08-01' existing = {
+resource managedCluster 'Microsoft.ContainerService/managedClusters@2022-06-01' existing = {
```

Comment:
For consistency I'd suggest to also update to the same version references to this resource type in other modules (e.g. the ones in modules\Microsoft.KubernetesConfiguration)

Reply:
Can't say I agree - but will do to get this PR in ;)

Reply:
done